content/v2.0/reference/cli/influx/_index.md

```diff
@@ -62,9 +62,15 @@ retrieving authentication tokens._
 | [user](/v2.0/reference/cli/influx/user)           | User management commands                             |
 | [write](/v2.0/reference/cli/influx/write)         | Write points to InfluxDB                             |
 
-{{% influx-cli-global-flags %}}
+## Mapped environment variables
+Some `influx` CLI flags are mapped to environment variables.
+If those environment variables are set, the flag automatically assumes the value
```

Comment:
Mapped flags get the value of the environment variable. To override this value, set the flag to another value.